lib/service/service.go

```diff
@@ -4008,9 +4008,13 @@ func (process *TeleportProcess) initProxyEndpoint(conn *Connector) error {
 			traceClt = clt
 		}
 
-		var accessGraphAddr string
+		var accessGraphAddr utils.NetAddr
```

Comment:
Could we keep this focused on python codegen and move other changes to a separate PR?